src/simplewallet/simplewallet.cpp

```diff
@@ -362,6 +362,29 @@ bool simple_wallet::seed_set_language(const std::vector<std::string> &args/* = s
   return true;
 }
 
+bool simple_wallet::change_password(const std::vector<std::string> &args)
+{
+  // watch-only wallets don't need passwords, right?
+  if(m_wallet->watch_only())
```

Comment:
I can't recall seeing any restrictions on passwords with watch only wallets. Was this something @moneromooo-monero mentioned?